checkin/src/store/index.js

```diff
@@ -71,29 +72,22 @@ const mutations = {
   [Methods.SET_FIXED_PRICE] (state, fixedPrice) {
     state.fixedPrice = fixedPrice
   },
-  [Methods.SET_MODAL] (state, status) {
-    state.activeModal = status
-  },
   [Methods.SET_RANDOM_SEAT] (state, seat) {
     state.seat = seat
     state.fixedPrice = 0
     state.seatPrice = 0
     state.totalPrice = 0
-  },
-  [Methods.SET_ACTIVITY] (state, status) {
-    state.isActive = status
   }
 }
 const actions = {
   selectSeat ({commit}, selectedSeat) {
-    commit(Methods.SET_ACTIVITY, true)
-    commit(Methods.SET_SELECTED_SEAT, selectedSeat)
+    commit(Methods.SET_DATA, 'SET_ACTIVITY', true)
```

Comment:
this should be `commit(Methods.SET_DATA, 'isActive', true)`. did you run this and try if it works?